great_expectations/datasource/data_connector/configured_asset_filesystem_data_connector.py

```diff
@@ -90,9 +93,11 @@ def _get_full_file_path_for_asset(
         base_directory: str = self.base_directory
         if asset is not None:
             if asset.base_directory:
-                base_directory = normalize_directory_path(
-                    dir_path=asset.base_directory,
-                    root_directory_path=base_directory,
+                base_directory = str(
```

Comment:
#dupe-01

Reply:
@donaldheppner This is V3 code and as per an earlier comment, we have to convert to `str`.  There may be an opportunity to move from `str` to typed objects in V3, but this is not in scope of the present effort (and I am also not sure how far this conversion would go, but that warrants an independent dedicated conversation).  Thank you.